websites/A/Apple TV/presence.ts

```diff
@@ -0,0 +1,187 @@
+class AppleTV extends Presence {
+  constructor(presenceOptions: PresenceOptions){
+    super(presenceOptions);
+  }
+
+  getVideo(){
+    return document.querySelector("apple-tv-plus-player")
+                    .shadowRoot.querySelector("amp-video-player-internal")
+                    .shadowRoot.querySelector("amp-video-player")
+                    .shadowRoot.querySelector("#apple-music-video-player") as HTMLVideoElement;
+  }
+
+  getTitle(eyebrow = false){
+    if (this.isWatching()){
+      const title = document.querySelector("apple-tv-plus-player")
+                            .shadowRoot.querySelector("amp-video-player-internal")
+                            .shadowRoot.querySelector("div.info__eyebrow")?.textContent;
+
+      if(title || eyebrow) return title;
+      else return document.querySelector("apple-tv-plus-player")
+                          .shadowRoot.querySelector("amp-video-player-internal")
+                          .shadowRoot.querySelector("div.info__title")?.textContent;
+    }
+    const title = document.querySelector("div.product-header__image-logo.clr-primary-text-on-dark > a > h2")?.textContent;
+
+    if (title) return title;
+    else return document.querySelector("img")?.alt;
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement
```suggestion
    else return document.querySelector<HTMLImageElement>("img").alt;
```